src/Express.re

```diff
@@ -381,6 +381,8 @@ module Response = {
   [@bs.send.pipe: t]
   external redirectCode : (int, string) => complete = "redirect";
   [@bs.send.pipe: t] external redirect : string => complete = "redirect";
+  [@bs.send.pipe: t] external set : (string, string) => t = "set";
```

Comment:
Don't these two functions do the same thing?
Would prefer a single function: `setHeader` maybe?